libbeat/common/kubernetes/k8skeystore/kubernetes_keystore.go

```diff
@@ -30,12 +31,13 @@ import (
 	"github.com/elastic/beats/v7/libbeat/logp"
 )
 
-type KubernetesKeystores map[string]keystore.Keystore
+//type KubernetesKeystores map[string]keystore.Keystore
+type KubernetesKeystores sync.Map
```

Comment:
Maybe you are looking for this, to keep the original methods:
```suggestion
type KubernetesKeystores = sync.Map
```
Or to hide the original type:
```suggestion
type KubernetesKeystores struct {
    sync.Map
}
```

Reply:
👍